src/utils/PolicyAdapter.ts

```diff
@@ -13,36 +13,39 @@ import { DeepPartial } from 'ts-essentials';
 
 export const toServerPolicy = (policy: DeepPartial<Policy>): ServerPolicyRequest => {
 
+    const { isEnabled, ...restPolicy } = policy;
+
     return {
-        ...policy,
-        is_enabled: policy.isEnabled, // eslint-disable-line @typescript-eslint/camelcase, camelcase
+        ...restPolicy,
+        is_enabled: isEnabled,
         actions: policy.actions?.map((action): string => {
             if (!action || !action.type) {
                 return '';
             }
 
-            let encodedAction = `${action.type} `;
+            let encodedAction = `${action.type}`;
 
             switch (action.type) {
                 case ActionType.WEBHOOK:
-                    encodedAction += action.endpoint;
+                    encodedAction += ' ' + action.endpoint;
```

Comment:
Are you sure that email is always 1st and webhook 2nd? Can't this be the other way around?

Reply:
I don't follow. Why are you asking if email is always 1st? Where is that implied?

Reply:
You mean the tests? They are processed in the order that the array has.

Reply:
I misread the code. You add the opt-arg for hooks after the space. And I read that you iterate over actions and add the action name.